setup.cfg

```diff
@@ -3,14 +3,16 @@
 # https://flake8.readthedocs.io/en/latest/user/configuration.html
 # https://flake8.readthedocs.io/en/latest/user/error-codes.html
 
-# Aligned with black https://github.com/psf/black/blob/main/.flake8
-extend-ignore = E203, E266, E501
```

Comment:
I think It's better to keep extend-ignore and remove the regular ignore

Reply:
I would be fine with this, one benefit I saw from keeping the `ignore` list and moving things there is that the default rules for `ignore` may change from version to version. By maintaining `ignore`, which currently maps to the defaults, we can effectively pin these. 

If we assume that whatever rules `flake8` selects as defaults are probably a good idea, I'd be more than happy to keep this rule-set even smaller.

Reply:
I'm fine either way. FWIW, I am usually an advocate for strict linting. I trust the two of you to find a good configuration

Reply:
Not sure where I got my earlier defaults for `flake8` from, but these are the actual ones: `'E226', 'E123', 'E24', 'E121', 'E126', 'E704', 'W503', 'W504'`. This means that a bunch of rules have changed. For comparison, `black` uses `ignore = E203, E266, E501, W503`. Given our previous selection, this feels like a reasonably strict set of rules and would mean that we mainly have to start fixing import formatting `E4**`.

Reply:
Silenced `E402` for tests.